charts/catalog/templates/apiregistration.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.useAggregator }}
-apiVersion: apiregistration.k8s.io/v1alpha1
+apiVersion: apiregistration.k8s.io/v1beta1
```

Comment:
@MHBauer I don't know of a trick, but it would have to but we can ask the user to specify it in another helm value.

Also, you could ask in the Kubernetes Slack #helm-dev channel to see if there's a trick